src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
-}
+@import './block/page';
+@import './block/stopwatch';
+@import './block/keyframes'
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), each import statement should be followed by a semicolon. Please add a semicolon at the end of line 3.